phpunit.xml.dist

```diff
@@ -28,6 +28,7 @@
 			<group>ms-files</group>
 			<group>ms-required</group>
 			<group>external-http</group>
+			<group>html-api-html5lib-tests</group>
```

Comment:
This should also be added to the multisite config located in `tests/phpunit/multisite.xml`

Reply:
asking as an ignorant party, why should this be run in the multisite suites as well? my assumption would be that since it doesn't depend or couple with any multisite code or context, that it wouldn't pair there in the tests.

Reply:
This is the exclusions, so adding it to the multisite config will mean it doesn’t run there by default as well

Reply:
Thanks, I've pushed a commit to add it to multisite as well.